compiler/damlc/daml-doc/src/DA/Daml/Doc/Extract/Templates.hs

```diff
@@ -200,8 +200,8 @@ isChoiceInst ctx inst
 
   | otherwise = Nothing
 
-hsExprsToString :: [HsExpr GhcPs] -> String
-hsExprsToString = intercalate ", " . fmap (showSDocUnsafe . ppr)
+hsExprToString :: HsExpr GhcPs -> String
+hsExprToString = showSDocOneLine unsafeGlobalDynFlags . ppr
```

Comment:
Moving forward, maybe we should do a review of doc generation and look at places where we could improve formatting. The way we currently pretty print signatories disallows the possibility of a signatory with a multiline definition - we simply compress the expression into a single line, which will probably make readability suffer pretty quickly.

Reply:
Agree, splicing out the source code by location would certainly be nicer